netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -397,14 +396,6 @@ public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
       ctx.pipeline().addBefore(ctx.name(), /* name= */ null, this.executor != null
           ? new SslHandler(sslEngine, false, this.executor)
           : new SslHandler(sslEngine, false));
-
-      // Support exporting the key and session identifier to the log named
-      // "io.netty.wireshark" when the system property named "io.netty.ssl.masterKeyHandler"
-      // is "true". This feature is used to analyze gRPC traffic with Wireshark.
-      if (Boolean.getBoolean(SslMasterKeyHandler.SYSTEM_PROP_KEY)) {
```

Comment:
Why this is deleted?

Reply:
Oh, it had some problem merging with the master. Fixing it now.